### DISCUSSION TITLE
I haven't get github education pack yet

### DISCUSSION THREAD

**Discussion by ShafiulKabirS** [community/community#144902] [GitHub Education]
### Hi everyone,

I applied for the github education student pack on 14th November 2024. That day the application was approved immediately. But haven't received the benefits yet. Today is 18th November 2024.
![Screenshot 2024-11-18 131557](https://github.com/user-attachments/assets/a29ca40d-86d9-45c1-be76-8491c70f4d5e)

**Comment by the-bipu** [comment]
It takes github at most 4 days to review your application as per [this](https://www.quora.com/How-long-does-it-take-to-be-approved-for-a-GitHub-Student-Developer-Pack) discussions of quora and i also agree with this as i've also waited for 3-4 days in my last edu account. you'll get it soon. 

Check out this discussion for more details:
[https://github.com/orgs/community/discussions/55559#discussioncomment-9795472](https://github.com/orgs/community/discussions/55559#discussioncomment-9795472)

**Comment by SubhashPavuskar** [comment]
use this link fill the form https://support.github.com/contact/education. you will get support from the community approvers.

**Comment by davevad93** [comment]
Hi @ShafiulKabirS ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ShafiulKabirS was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
